Can't import matplotlib

<!--To help us understand and resolve your issue, please fill out the form to the best of your ability.-->
<!--You can feel free to delete the sections that do not apply.-->

### Bug report

**Bug summary**

Importing matplotlib fails with `ImportError: cannot import name 'get_backend' from 'matplotlib'`.

**Code for reproduction**

<!--A minimum code snippet required to reproduce the bug, also minimizing the number of dependencies required-->

```python
import matplotlib
```

**Actual outcome**

<!--The output produced by the above code, which may be a screenshot, console output, etc.-->

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.7/site-packages/matplotlib/__init__.py", line 1080, in <module>
    rcParamsOrig = RcParams(rcParams.copy())
  File "/usr/lib/python3.7/site-packages/matplotlib/__init__.py", line 870, in __getitem__
    from matplotlib import pyplot as plt
  File "/usr/lib/python3.7/site-packages/matplotlib/pyplot.py", line 33, in <module>
    import matplotlib.image
  File "/usr/lib/python3.7/site-packages/matplotlib/image.py", line 18, in <module>
    from matplotlib.backend_bases import FigureCanvasBase
  File "/usr/lib/python3.7/site-packages/matplotlib/backend_bases.py", line 46, in <module>
    from matplotlib import (
ImportError: cannot import name 'get_backend' from 'matplotlib' (/usr/lib/python3.7/site-packages/matplotlib/__init__.py)
```

**Expected outcome**

Matplotlib is imported

**Matplotlib version**
<!--Please specify your platform and versions of the relevant libraries you are using:-->
  * Operating system: GNU/Linux
  * Matplotlib version: 3.0 git master
  * Python version: 3.7.1

Python from Arch repo, matplotlib from git master.

I can confirm by building from master branch in a clean chroot, during the tests part:
```
E   _pytest.config.ConftestImportFailure: (local('/build/python-matplotlib-git/src/matplotlib-test/lib/matplotlib/tests/conftest.py'), (<class 'ImportError'>, ImportError("cannot import name 'get_backend' from 'matplotlib' (/build/python-matplotlib-git/src/matplotlib-test/lib/matplotlib/__init__.py)"), <traceback object at 0x7f591217ff08>))
```

--- a/lib/matplotlib/__init__.py
+++ b/lib/matplotlib/__init__.py
@@ -924,6 +924,10 @@ class RcParams(MutableMapping, dict):
                         for key, value in self.items()
                         if pattern_re.search(key))
 
+    def copy(self):
+        return {k: dict.__getitem__(self, k)
+                for k in self}
+
 
 def rc_params(fail_on_error=False):
     """Return a :class:`matplotlib.RcParams` instance from the

